paper/main.md

```diff
@@ -55,7 +55,9 @@ Furthermore, library developers will have the chance to see how their products a
 
 ## Introduction
 
-_Here we can talk about the ATLAS experiment, the detector, its purpose, the amount of data collected, etc._
+How can the contents of `paper/introduction.md` get inserted here in a similar manner to an include directive in LaTeX?
+
+It is unclear how to get myst to build multiple files into a single view like shown in jupyterbook: https://jupyterbook.org/en/stable/structure/toc.html#build-an-article-from-multiple-files as there is no `--builder` option for `myst start` or `myst build`.
```

Comment:
@rowanc1 while I'll ask elsewhere as well (probably Discord or Discussions), as this is SciPy 2024 related, can you confirm that what I want to do is possible and if so, point to docs on how?

Reply:
Asked as https://github.com/orgs/executablebooks/discussions/1182

Reply:
Resolved by @agoose77 in https://github.com/orgs/executablebooks/discussions/1182#discussioncomment-9540467!

Reply:
Hi @matthewfeickert -- just got this notification now (was on vacation), glad that you are through this!